src/player/APVideoPlayer.js

```diff
@@ -25,7 +25,7 @@ const APReactVideoView = {
   name: 'ReactVideoView',
   propTypes: {
     src: PropTypes.shape({
-      id: PropTypes.string,
+      id: PropTypes.number,
```

Comment:
Wouldn't it be better to use strings instead? As this is a reusable component, other players may use id's that aren't numbers.

Reply:
Native complains that it wants a double

Reply:
Alright, so it looks like aligning the current implementation. Shouldn't be a problem to refactor when necessary. LGTM